src/ng/parse.js

```diff
@@ -401,7 +400,7 @@ AST.prototype = {
 
   program: function() {
     var body = [];
-    while (true) {
+    for (;;) {
```

Comment:
Globally. I don't like such exceptions, this code is not special so if
`while (true)` here is fine, I'd enable it everywhere.

I mostly cared about disallowing stuff like `if (false)`. :)

On Fri, Aug 5, 2016 at 4:40 PM, Martin Staffa <notifications@github.com>
wrote:

> In src/ng/parse.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r73702677>:
>
> > @@ -401,7 +400,7 @@ AST.prototype = {
> >
> >    program: function() {
> >      var body = [];
> > -    while (true) {
> > +    for (;;) {
>
> Just for this line or globally?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.js/pull/14952/files/4ab2433bb71fe2bf692ca169c46363385a3806e4#r73702677>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnhSjWj5tpkaIUXp2MgMPMdxczppyks5qc0tFgaJpZM4JU90D>
> .
>



-- 
Michał Gołębiowski